packages/components/organisms/f-checkout/CHANGELOG.md

```diff
@@ -3,13 +3,24 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+
+v0.152.0
+------------------------------
+*July 6, 2021*
+
+### Added
+- Experiment headers to PATCH checkout request
+- Track experiments if returned in response headers
+
+
 v0.151.0
 ------------------------------
 *July 5, 2021*
 
 ### Fixed
 - Get new `availableFulfilment` times when no available fulfilment times are available.
 
+
```

Comment:
Super nit: This line isn't necessary

Reply:
I thought we kept 2 lines between each addition to the changelog so spotted that and added it 😅

Reply:
We do, but only between separate version entries, not within them 😄

Reply:
Shows how much of a state I am that I didn't even notice that 😂  - updated